### ISSUE TITLE
Resarch CIAS Card Games

### ISSUE THREAD

**Issue by edogdu** [edogdu/RAM_Cyber_Defenders#2]

**Comment by Luk-Er4** [comment]
First of all, I researched the Cyber Threat Guardian card game. There are three versions of the game, Grades K-1, Grades 1-2, and Grades 2 and up. I will go over the Grades 2 and up version.

The interesting feature of this game is that the logic to make this game is not complicated compared to any other one. It is because there are only four possible events when players take turns: draw a card, put a card on the board, remove the opponent's card, and discard a card. 
The other thing is that blue, green, and red cards can be activated depending on their categories: house icon, lock icon, shield icon, and wireless icon. This feature exists only in this game, not in the other two games but still, it does not make it hard to build logic.

Here is the attachment of a flowchart on this game.

![Cyber Threat Guardian](https://github.com/edogdu/RAM_Cyber_Defenders/assets/102909145/95da041b-1fde-483c-afeb-cde0561da262)

**Comment by edogdu** [comment]
> First of all, I researched the Cyber Threat Guardian card game. There are three versions of the game, Grades K-1, Grades 1-2, and Grades 2 and up. I will go over the Grades 2 and up version.
> 
> The interesting feature of this game is that the logic to make this game is not complicated compared to any other one. It is because there are only four possible events when players take turns: draw a card, put a card on the board, remove the opponent's card, and discard a card. The other thing is that blue, green, and red cards can be activated depending on their categories: house icon, lock icon, shield icon, and wireless icon. This feature exists only in this game, not in the other two games but still, it does not make it hard to build logic.
> 
> Here is the attachment of a flowchart on this game and I will comment on Cyber Threat Protector.
> 
> ![Cyber Threat Guardian](https://private-user-images.githubusercontent.com/102909145/299501219-95da041b-1fde-483c-afeb-cde0561da262.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDYxNDM5ODYsIm5iZiI6MTcwNjE0MzY4NiwicGF0aCI6Ii8xMDI5MDkxNDUvMjk5NTAxMjE5LTk1ZGEwNDFiLTFmZGUtNDgzYy1hZmViLWNkZTA1NjFkYTI2Mi5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEyNVQwMDQ4MDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05YTAyOWUwNWJmZjM0ZjE0ZjY5Y2Y2ZmYwMDZjMTAwNDFiODE1MTMzNmQzYzcwOTIyMzZlYzUzMzUwYjkyN2VkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.StjEX4UcD2jgKz_IbuU6Xd1Hu0q8UYKKku5OeweqGwQ)

Eujin, this is great! Thank you for the detailed explanation. I agree with your selection. Thanks!

**Comment by LightTheBlues** [comment]
I installed an online version of Cyber Threat Defender. The reason I chose Cyber Threat Defender is that it seems to be the most popular game from CIAS, with tournaments, challenges, and apparent popularity among students. I also considered it easier to learn as it has a free digital version of the game.

The rules seem simple: blue cards give you points, green cards protect your points, red cards decrease your opponents' points, and yellow cards have special effects. The rules appear straightforward and easy to understand.

I don't have diagram like Eujin does, and I am fine implementing either one of them.
